Reference/Searching/Examine/index.md

```diff
@@ -20,6 +20,10 @@ Get up and running with Examine straight away with this quick start guide.
 
 Learn how to create your own Lucene indexes using Examine in Umbraco 8.
 
+## [PDF indexing and multisearchers](pdf-index.md)
+
+Learn how to index PDF files in Examine and how to create a multisearcher that searches through both the externalindex and the pdfindex.

```

Comment:
Wondering whether _externalindex_ and _pdfindex_ should be written in a different way?
Perhaps something like _External Index_ and _PDF Index_? 🤔